taskcluster/fenix_taskgraph/target_tasks.py

```diff
@@ -44,7 +43,13 @@ def target_tasks_nightly_on_google_play(full_task_graph, parameters, graph_confi
     """Select the set of tasks required for a nightly build that goes on Google Play."""
 
     def filter(task, parameters):
-        return task.attributes.get("nightly", False)
+        return (
+            task.attributes.get("nightly", False) and
+            # This target_task is temporary while Google Play processes APKs slower than usually
+            # (bug 1628413). So we want this target task to be only about shipping APKs to GP and
+            # not doing any other miscellaneous tasks like performance testing
+            task.kind not in ('browsertime', 'visual-metrics', 'raptor')
```

Comment:
Small nit about using `'` here and `"` in the previous condition for `"nightly"`

Reply:
👍 I'm not super good at remaining consistent with these. Tools like [black](https://pypi.org/project/black/) are really good at reminding me to use only one of them. We don't have it installed on this project, but that's probably a good addition to have.